<launch>

    <arg name="vehicle_name" />
    <arg name="pos_x" />
    <arg name="pos_y" />
    <arg name="pos_z" />


    <group ns="$(arg vehicle_name)">
        <param name="pos_x" value="$(arg pos_x)"/>
        <param name="pos_y" value="$(arg pos_y)"/>
        <param name="pos_z" value="$(arg pos_z)"/>

        <node name="Beer_pos" pkg="gripper" type="beer_publisher.py"/>
        <node name="pathfinder" pkg="gripper" type="pathfinder.py" output="screen" clear_params="true"/>

        <!--node name="BEER" pkg="gazebo_ros" type="spawn_model"
        args="-sdf -file $(find gripper)/models/beer2/model.sdf
              -model beer
              -x $(arg pos_x) -y $(arg pos_y) -z $(arg pos_z)
              -R 0 -P 0 -Y 0" /-->
            
    </group>
</launch>